### ISSUE TITLE
Emoji gender

### ISSUE THREAD

**Issue by amivaleo** [Maestroschan/emoji-selector-for-gnome#45]
Hi :)

I saw you put a gender button in the "people & clothing" category, but:
- both genders are shown (I would expect to see only one emoji per type, which changes according to the selected gender)
- both buttons are not effective
- if I click twice on the same button, it gets inactive. It should remain selected. I'd say more: one of the two buttons should always be on and the other should stay off

I suppose you're still working on this feature, thus they're not complete, I guess.

Thank you for your efforts. :1st_place_medal:

**Comment by Maestroschan** [comment]
It might not be clear because it reflects the technical construction of emojis. There are 2 kind of gendered emojis in the unicode standard:

For example:

- 👨‍🌾 is "man farmer", and is done with "man"+u200d+"ear of rice"
- 👩‍🌾is "woman farmer", and is done with "woman"+u200d+"ear of rice"

In this case, both are always shown, and the button has no effect.

But in the other hand:

- 🙋 is "person raising hand", without gender indication from the unicode standard
- 🙋‍♂️ is "man raising hand" ("person raising hand"+u200d+"male sign"+ufe0f)
- 🙋‍♀️ is "woman raising hand" ("person raising hand"+u200d+"female sign"+ufe0f)

This is the category of emojis which is impacted by the "gender" button

>if I click twice on the same button, it gets inactive

Being able to have no selected gender is not a bug: it will produce the first one (the gender will only depend on the font designers choices). It can be useful if the system can't render a compound emoji.

>both buttons are not effective

The displayed emojis don't change (for now), but the copied emoji should be impacted if it belongs to the 2nd category

**Comment by amivaleo** [comment]
> There are 2 kind of gendered emojis in the unicode standard
I seee. :(
That's misleading. But it doesn't depend on you. I see your point here.

>  the copied emoji should be impacted if it belongs to the 2nd category
Yes, yes! That works.
I'm sorry, I only checked a few emojis of the "first" type.

Thank you. :)

**Comment by AlexWayfer** [comment]
Also, there is an issue with search:

1. I've picked male gender (is blue background for selected?):

![image](https://user-images.githubusercontent.com/6510020/66406449-4c207b80-e9f4-11e9-971d-e40e83a5fe78.png)

2. I'm getting an emoji in the selected gender: 🤷🏻‍♂️
 
3. But in search there is default (without?) gender:

![image](https://user-images.githubusercontent.com/6510020/66406347-27c49f00-e9f4-11e9-892e-4bdb4fa777ee.png)

🤷

**Comment by maoschanz** [comment]
yes probably i began something else in the middle of the implementation, i'll come back to this later
